wordpress-mobile/WordPress-iOS
# Page the requests for media on XMLRPC.

Fixes #8932 

This PR implements the paging for the requests of the Media Library of WP.org blog using the XMLRPC Api.

Instead of requesting the library in one go we request pages with 100 elements each.

The way the XMLRPC api implements paging using offset and page could cause some minor data sync issues if media is added to the library at the exact time a sync is happening, but this should auto correct itself on the next sync and was considered a minor issue comparing to server crashes returning big media libraries.

To test:
 - Start the app
 - Add a WP.org site to the site list that **does not** use Jetpack
 - Open the Media Library
 - Check if all media items are there
 - Check with an empty media library just to be sure
